actors/abi/bitfield.go

```diff
@@ -0,0 +1,232 @@
+package abi
+
+import (
+	"errors"
+	"fmt"
+	"io"
+
+	cbg "github.com/whyrusleeping/cbor-gen"
+	"golang.org/x/xerrors"
+
+	rlepluslazy "github.com/filecoin-project/specs-actors/actors/util/rleplus"
+)
+
+var ErrBitFieldTooMany = errors.New("to many items in RLE")
+
+type BitField struct {
+	rle rlepluslazy.RLE
+
+	bits map[uint64]struct{}
+}
+
+func NewBitField() BitField {
+	bf, err := NewBitFieldFromBytes([]byte{})
+	if err != nil {
+		panic(fmt.Sprintf("creating empty rle: %+v", err))
+	}
+	return bf
+}
+
+func NewBitFieldFromBytes(rle []byte) (BitField, error) {
+	bf := BitField{}
+	rlep, err := rlepluslazy.FromBuf(rle)
+	if err != nil {
+		return BitField{}, xerrors.Errorf("could not decode rle+: %w", err)
+	}
+	bf.rle = rlep
+	bf.bits = make(map[uint64]struct{})
+	return bf, nil
+
+}
+
+func BitFieldFromSet(setBits []uint64) BitField {
+	res := BitField{bits: make(map[uint64]struct{})}
+	for _, b := range setBits {
+		res.bits[b] = struct{}{}
+	}
+	return res
+}
```

Comment:
This method is unused in Lotus and here. @magik6k can we drop it?

Reply:
Sounds good

Reply:
Now using this method here: https://github.com/filecoin-project/specs-actors/pull/85/files#diff-3718ef74dd0a657e370fc4ee95eb337fR127